sky/provision/gcp/constants.py

```diff
@@ -103,6 +105,29 @@
     '--device /dev/nvidia-uvm:/dev/nvidia-uvm',
     '--device /dev/nvidiactl:/dev/nvidiactl',
     '--env LD_LIBRARY_PATH=/usr/local/nvidia/lib64:/usr/local/tcpx/lib64',
+    '--env NCCL_GPUDIRECTTCPX_SOCKET_IFNAME=eth1,eth2,eth3,eth4',
+    '--env NCCL_GPUDIRECTTCPX_CTRL_DEV=eth0',
+    '--env NCCL_GPUDIRECTTCPX_TX_BINDINGS="eth1:8-21,112-125;eth2:8-21,112-125;eth3:60-73,164-177;eth4:60-73,164-177"',
+    '--env NCCL_GPUDIRECTTCPX_RX_BINDINGS="eth1:22-35,126-139;eth2:22-35,126-139;eth3:74-87,178-191;eth4:74-87,178-191"',
+    '--env NCCL_GPUDIRECTTCPX_PROGRAM_FLOW_STEERING_WAIT_MICROS=50000',
+    '--env NCCL_GPUDIRECTTCPX_UNIX_CLIENT_PREFIX="/run/tcpx"',
+    '--env NCCL_GPUDIRECTTCPX_FORCE_ACK=0',
+    '--env NCCL_SOCKET_IFNAME=eth0',
+    # '--env NCCL_CROSS_NIC=0',
+    # '--env NCCL_ALGO=Ring',
+    # '--env NCCL_PROTO=Simple',
+    # '--env NCCL_NET_GDR_LEVEL=PIX',
+    # '--env NCCL_DYNAMIC_CHUNK_SIZE=524288',
+    # '--env NCCL_P2P_NET_CHUNKSIZE=524288',
+    # '--env NCCL_P2P_PCI_CHUNKSIZE=524288',
+    # '--env NCCL_P2P_NVL_CHUNKSIZE=1048576',
+    # '--env NCCL_BUFFSIZE=8388608',
+    # '--env NCCL_MAX_NCHANNELS=8',
+    # '--env NCCL_MIN_NCHANNELS=8',
+    # '--env NCCL_NSOCKS_PERTHREAD=4',
+    # '--env NCCL_P2P_PXN_LEVEL=0',
+    # '--env NCCL_SOCKET_NTHREADS=1',
+    # '--env CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7',
```

Comment:
Should we just remove this here and update our NCCL test to only include hte NCCL related env vars?

Reply:
Removed. It's already in the NCCL tests